src/modules/logger/logger.service.ts

```diff
@@ -54,14 +35,31 @@ export class Logger extends NestLogger implements LoggerService {
       },
     });
 
-    const console = new transports.Console({ format: logFormat });
+    const logFormat = format.combine(
+      format.timestamp(),
+      format.printf(
+        ({ level, message, timestamp, context }) =>
+          `${level} [${context || ''}] : ${timestamp} - ${this.redactor.redact(
+            message,
+          )}`,
+      ),
+      format.colorize(),
+    );
 
-    const transport = isProduction ? file : console;
+    const file = new transports.DailyRotateFile({
+      filename: 'iam-cache-server-%DATE%.log',
+      zippedArchive: true,
+      format: logFormat,
+      dirname: configService.get<string>('LOGS_DIRECTORY'),
+      maxFiles: '14d',
+    });
+
+    const console = new transports.Console({ format: logFormat });
 
     this.logger = createLogger({
       format: logFormat,
       level: 'debug',
-      transports: [transport],
+      transports: [console, file],
```

Comment:
@Passerino this means that we are logging to both `console` and `file` then?

Reply:
yes, I don't think we will look at the file log anytime soon, but wanted to keep backward compatibility.

Reply:
@hejkerooo do you see any issue with logging to a file in a kubernetes pod?

Reply:
it's a rolling file so I don't foreseen any storage limit issues.